README.md

```diff
@@ -63,64 +89,103 @@ Untracked files:
+`README.md` to the repository and check the status, we would type:
 
 ```
-next-big-thing $ git add README.md
-next-big-thing $ git status
+my-git-project $ git add README.md
+my-git-project $ git status
 On branch master
 
-Initial commit
+No commits yet
 
 Changes to be committed:
   (use "git rm --cached <file>..." to unstage)
 
 	new file:   README.md
 ```
 
-You can now see that Git is ready to keep track of `README.md` and recognizes that the file is new to this repository. However, while we've informed Git that there is a new file, we still haven't informed Git that this new file is considered a change to our repository. We create changes in our repository by making "commits."
+We can now see that Git is ready to keep track of `README.md` and recognizes
+that the file is new to this repository. However, while we've informed Git that
+there is a new file, we still haven't informed Git that this new file is
+considered a change to our repository. We create changes in our repository
+by making "commits."
 
 **To capture all changes in a directory –– the standard way to do it –– type `git add .`, where the `.` refers to the entire current directory.**
 
-## Cataloging a new version of your code with `git commit`
+## Create a Commit With `git commit` and Apply a Commit Message
 
-Git allows us to mark changes to our code as different versions, or, in Git speak, "commits." A commit is like a frozen copy of your code at a given point. Once you've made a commit, you can always easily revert to that version of your code as it existed at that exact moment.
+Commits are amazingly powerful in `git`. With a commit you can move your project
+back to a past commit and throw away bad ideas. You can tell git to create a new
+"parallel universe" based off of a past commit and you can start working in that
+universe and decide later to keep that work or throw it away. We won't cover
+these features in this lesson, but they exist, and are very useful.
 
-Now that Git is aware of a change to our project –– the addition of `README.md` –– let's submit the first official version of our project using `git commit`. Whenever we make a commit, we must supply a commit message describing the version or commit. This message log makes it easy for us to figure out what each commit or version is all about.
+Now that Git is aware of a change to our project –– the addition of `README.md`
+–– let's submit the first official version of our project using `git commit`.
+Whenever we make a commit, we must supply a commit message describing the version
+or commit. This message log makes it easy for us to figure out what each commit
+or version is all about.
 
-To make your first commit, type: `git commit -m "Added README.md"`. This tells Git that our commit message, represented by the `-m` flag, is `"Added README.md"`.
+To make our first commit, type: `git commit -m "Initial commit"`. This tells
+Git that our commit message, represented by the `-m` flag, is `"Initial commit"`.
```

Comment:
> It knows that the state of `README.md` is to be recorded as part of this commit because our last use of `git status` listed `README.md` under `Changes to be committed:`. That's why a _commit_ is composed of two steps: adding the files in the snapshot (_commit_) and then doing the commit with your message.